### PR TITLE
Update git setup for Github as official upstream

### PR DESCRIPTION
**Pull request by blowekamp**

### CODE REVIEW

**Review by dave3d — APPROVED** [review]
Looks awesome!

File: Utilities/SetupForDevelopment.sh [hunk 1]
```diff
@@ -80,29 +80,37 @@ echo -e "Git version $git_version is OK.\n"
 
 cd "${BASH_SOURCE%/*}/.." &&
```
Comment (zivy) [inline Utilities/SetupForDevelopment.sh]: For some reason cd "${BASH_SOURCE%/*}/.." didn't work on my mac (using bash). When I ran the script from inside the Utilities directory I get the error: "./SetupForDevelopment.sh: line 82: Utilities/GitSetup/setup-user: No such file or directory"

It does work when running from the directory above Utilities.

**Review by zivy — COMMENTED** [review]
Had some issues with the setup. They do not appear to be critical as the origin and upstream repos were set correctly and when I do git config --list --show-origin and see what was set in the SimpleITK repo things look reasonable.

File: Utilities/SetupForDevelopment.sh (hunk 1 above)
Comment (blowekamp) [reply Utilities/SetupForDevelopment.sh]: This second `cd` is not needed, and is broken with relative paths. Removed in next patch.

**Pull request merged**